chrome/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "chrome",
+  "version": "0.4.0",
+  "description": "Click to report an incompatible site at webcompat.com, and optionally upload a screenshot.",
+  "author": "Mike Taylor",
```

Comment:
You don't have to change this, but at this point "Mike Taylor" as author doesn't make much sense. ^_^

Maybe something like "webcompat.com team"?